sfdx-source/apex-mocks/test/classes/fflib_IDGeneratorTest.cls

```diff
@@ -52,5 +52,10 @@ private class fflib_IDGeneratorTest
 		System.assertEquals('001000000000009AAA', id9);
 		System.assertEquals('001000000000010AAA', id10);
 		System.assertEquals('001000000000011AAA', id11);
+
+		//	Given History SObjectTypes
```

Comment:
If we decide to implement a configuration option for the 'History' and '017' key prefix, let's implement that here as well.

Reply:
so -- a CMT solution (like a regex for the pattern to identify xxxHistory) and a keyPrefix is certainly doable - it would introduce for the first time a CMT in this repo and wouldn't necessarily deal with all wild futures like custom objects getting one history key prefix versus standard objects getting a diff key prefix.  Of course what we'd want to see is all xxxHistory getting unique key prefixes so result would come back in describe. I wonder if SFDC did it this way to preserve the range of available key prefixes reserved for standard objects

@stohn777